src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/ConvertToJsonCommand.cs

```diff
@@ -64,6 +64,15 @@ public class ConvertToJsonCommand : PSCmdlet
         [Parameter()]
```

Comment:
Could you please fix the line too and use the file pattern `[Parameter] `?

Reply:
Fixed.